### ISSUE TITLE
Iteration Plan for 0.0.2

### ISSUE THREAD

**Issue by ARudiuk** [abrahamnunes/fitr#65]
This plan captures the work that needs to be done before 0.0.2 of the package gets released. 
The planned release date for this is late July.

This iteration will involve a complete re-work of the overall structure of fitr, along with a change in what tools will be provided, and where future work will be focused. This change is being made to reflect a more modular nature that is going to be taken with fitr. This means less off a focus on providing pre-made models, and rather providing more tools to build, visualize, fit, and test models. 

Actions of pypi release will be covered in issue ...

## Plan Items
I am following a similar structure to vscode. Below is a summary of the top level plan items.

Legend of annotations

Mark | Description
-- | --
🏃 | work in progress
✋ | blocked task
💪 | stretch goal for this iteration
🔴 | missing issue reference
🔵 | more investigation required to remove uncertainty
⚫️ | under discussion within the team

### fitr structure

- [ ] 🔴Replace fitr code 
    - [ ] 🔴Remove unused fitr code
    - [ ] 🔴Add new fitr code @abrahamnunes 
- [ ] 🔴Update README
    - [ ] 🔴Remove Unused Tags
    - [ ] 🔴Change to standardized open source template
    - [ ] 🔴Update to markdown from rst
- [ ] 🔴Change what's_new to md from rst
- [ ] 🔴Update CONTRIBUTING
    - [ ] 🔴Change to standardized open source template
- [ ] 🔴Check LICENSE
- [ ] Clear old issues or re-word #10 #11 #32 #34 #60
- [ ] 🔴Potentially remove _config.yml 
- [ ] 🔴Update requirements.txt
- [ ] 🔴Potentially remove requirements_docs
- [ ] 🔴Investigate purpose of requirements_test

### pypi Release
- [ ] 🔴Do a pass through setup.py
- [ ] 🔴Potentially remove setup.cfg if not needed anymore

### Documentation
- [ ] 🔴Remove old documentation
- [ ] 🔴 Find new documentation method

### Example
- [ ] 🔴 Remove old examples

### Testing
- [ ] 🔴 Remove old tests
- [ ] 🔴Update Travis to pass

### Deferred Items
- [ ] 🔴Tests for new code
- [ ] 🔴New examples
